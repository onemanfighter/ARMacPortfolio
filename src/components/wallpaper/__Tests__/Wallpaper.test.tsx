import { WallpaperEnum } from '@settingsStore';
import { render } from '@testing-library/react';
import WallpaperComponent from '../Wallpaper';

describe('Wallpaper component', () => {
  it('should return default wallpaper type 1', () => {
    const { container } = render(
      <WallpaperComponent id={WallpaperEnum.Wallpaper1} />,
    );

    expect(container).toMatchSnapshot();
  });

  it('should return wallpaper for wallpaper 2', () => {
    const { container } = render(
      <WallpaperComponent id={WallpaperEnum.Wallpaper2} />,
    );

    expect(container).toMatchSnapshot();
  });

  it('should return wallpaper for wallpaper 3', () => {
    const { container } = render(
      <WallpaperComponent id={WallpaperEnum.Wallpaper3} />,
    );

    expect(container).toMatchSnapshot();
  });

  it('should return wallpaper for wallpaper 4', () => {
    const { container } = render(
      <WallpaperComponent id={WallpaperEnum.Wallpaper4} />,
    );

    expect(container).toMatchSnapshot();
  });

  it('should return wallpaper for wallpaper 5', () => {
    const { container } = render(
      <WallpaperComponent id={WallpaperEnum.Wallpaper5} />,
    );

    expect(container).toMatchSnapshot();
  });

  it('should return wallpaper for wallpaper 6', () => {
    const { container } = render(
      <WallpaperComponent id={WallpaperEnum.Wallpaper6} />,
    );

    expect(container).toMatchSnapshot();
  });

  it('should return wallpaper for wallpaper 7', () => {
    const { container } = render(
      <WallpaperComponent id={WallpaperEnum.Wallpaper7} />,
    );

    expect(container).toMatchSnapshot();
  });
});
