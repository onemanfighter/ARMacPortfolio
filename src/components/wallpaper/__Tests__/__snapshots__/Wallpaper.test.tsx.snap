// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wallpaper component should return default wallpaper type 1 1`] = `
<div>
  <img
    aria-label="wallpaper-component-with-wallpaper1"
    class="chakra-image css-uwwqev"
    loading="lazy"
    src=""
  />
</div>
`;

exports[`Wallpaper component should return wallpaper for wallpaper 2 1`] = `
<div>
  <img
    aria-label="wallpaper-component-with-wallpaper2"
    class="chakra-image css-uwwqev"
    loading="lazy"
    src=""
  />
</div>
`;

exports[`Wallpaper component should return wallpaper for wallpaper 3 1`] = `
<div>
  <img
    aria-label="wallpaper-component-with-wallpaper3"
    class="chakra-image css-uwwqev"
    loading="lazy"
    src=""
  />
</div>
`;

exports[`Wallpaper component should return wallpaper for wallpaper 4 1`] = `
<div>
  <img
    aria-label="wallpaper-component-with-wallpaper4"
    class="chakra-image css-uwwqev"
    loading="lazy"
    src=""
  />
</div>
`;

exports[`Wallpaper component should return wallpaper for wallpaper 5 1`] = `
<div>
  <img
    aria-label="wallpaper-component-with-wallpaper5"
    class="chakra-image css-uwwqev"
    loading="lazy"
    src=""
  />
</div>
`;

exports[`Wallpaper component should return wallpaper for wallpaper 6 1`] = `
<div>
  <img
    aria-label="wallpaper-component-with-wallpaper6"
    class="chakra-image css-uwwqev"
    loading="lazy"
    src=""
  />
</div>
`;

exports[`Wallpaper component should return wallpaper for wallpaper 7 1`] = `
<div>
  <img
    aria-label="wallpaper-component-with-wallpaper7"
    class="chakra-image css-uwwqev"
    loading="lazy"
    src=""
  />
</div>
`;
